roles/express-api/templates/start-express-container.sh.j2

```diff
@@ -0,0 +1,51 @@
+api_name=$1
```

Comment:
Any reason why this file is a template?

Reply:
Good catch :stuck_out_tongue:. Will change to a normal file.

Reply:
Done.